server/lib/model/rule.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+
+import '../proto/rule.pb.dart' as proto;
+import '../service/dv360.dart';
+import 'action.dart';
+
+/// A class that represents a rule to manipulate DV360 entities.
+class Rule {
+  /// An [Action] instance that manipulates DV360 entities.
+  final Action _action;
+
+  /// Creates a [Rule] from a [proto.Rule].
+  Rule.fromProto(proto.Rule protoRule)
+      : _action = getActionFromProto(protoRule.action);
```

Comment:
Does it need to change to `Action.getActionFromProto()`?

Reply:
Completely forgot 😅